pkg/common/util/config.go

```diff
@@ -17,21 +17,29 @@ limitations under the License.
 package util
 
 import (
+	"io/ioutil"
 	"os"
 	"path"
 	"path/filepath"
 	"strings"
 
 	"github.com/ServiceComb/go-archaius/sources/file-source"
+	"gopkg.in/yaml.v2"
```

Comment:
Please sort the import packages in alphabetical order.

Reply:
Dujun, here i have separated the  standard go packages, vendor packages and kubeedge packages. They are already sorted in alphabetical order within themselves.

Reply:
I mean:

```
github.com/kubeedge/kubeedge/beehive/pkg/common/config
github.com/ServiceComb/go-archaius/sources/file-source

gopkg.in/yaml.v2
```